src/components/SearchBox/__tests__/SearchBox-test.js

```diff
@@ -153,14 +156,19 @@ describe('SearchBox', () => {
       });
     });
 
-    test('sets focus with autofocus to true', () => {
+    // @TODO Since the Preact X migration and new testing environment, this test
+    // doesn't pass.
+    // eslint-disable-next-line jest/no-disabled-tests
+    test.skip('sets focus with autofocus to true', () => {
```

Comment:
can you try changing `autoFocus` to `autofocus` on the component?

Reply:
It doesn't work either.

Reply:
Seems to be a limitation of `testing-library`/`jsdom` (not sure, I did not find a lot of information on `autofocus`). I will open an issue for this.

I think that right now the only thing we can test is the presence of the `autofocus` attribute on the element:

```js
expect(input).toHaveAttribute('autofocus', 'true');
```

Reply:
That's a good temporary workaround!